installer.sh

```diff
@@ -328,8 +328,19 @@ install_package_macos() {
 	fi
 }
 
+# Tests if the given port is in use
+is_port_in_use() {
+	lsof -Pi :"$1" -sTCP:LISTEN -t > /dev/null
+}
+
 print_bold "Welcome to the ioBroker installer!" "Installer version: $INSTALLER_VERSION" "" "You might need to enter your password a couple of times."
 
+if [ is_port_in_use 9000 ] || [ is_port_in_use 9001 ]; then
+	echo "${yellow}Port 9000 or 9001 is in use. This could mean that ioBroker is already running."
```

Comment:
maybe better:
Port 9000 or 9001 is in use by an other process. Please use 'netstat -tulpn' to check which process is using these ports.

Reply:
I think we should actually list which process it is.

Reply:
This would be even better.
Linux:

```
ingof@io1:~$ sudo netstat -tulpn|grep 9001
tcp        0      0 0.0.0.0:9001            0.0.0.0:*               LISTEN      1548/iobroker.js-co 
```

Important: macos geht wieder mal anders...

```
GF0586MAC:iobroker ingof$ sudo lsof -iTCP:9001 -sTCP:LISTEN 
COMMAND   PID  USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME
node    23594 ingof   23u  IPv4 0x969ad6bd32f95965      0t0  TCP localhost:etlservicemgr (LISTEN)
```

Nicht so schön übersichtlich